src/hooks/useContract.ts

```diff
@@ -33,6 +34,32 @@ function useContract(address: string | undefined, ABI: any, withSignerIfPossible
   }, [address, ABI, library, withSignerIfPossible, account]);
 }
 
+/**
+ * This function is used to return ethers contract instance without isAddress check because some address from Hedera is not valid Ethereum address
+ * @param address address of contract
+ * @param ABI ABI of contract
+ * @param withSignerIfPossible whether to use signer (if it exists) to sign messages or not
+ * @returns Return's Ethers Contract Instance
+ */
+function useHederaContract(address: string | undefined, ABI: any, withSignerIfPossible = true): Contract | null {
```

Comment:
I am little bit confused, why do we need `useHederaContract` because till now I haven't encountered any situation where we can't use `useContract`